File: Python/Base/Sort/SortByIndex.py
arr: list[int] = [43,5,23,14,52,32,90]

for i in range(0, len(arr) -1):
    minimumNumberIndex : int = i
    for j in range( i + 1, len(arr)):
        if arr[j] < arr[minimumNumberIndex]:
            minimumNumberIndex = j
    
    temp: int = arr[i]
    arr[i] = arr[minimumNumberIndex]
    arr[minimumNumberIndex] = temp
    

print(arr)

File: TypeScript/Base/Sort/SortByIndex.ts
let arrToSort: number[] = [20,31,3,45,65,23,12]

for (let i = 0; i < arrToSort.length -1; i++) {
   let minimumNumberIndex : number = i
   for (let j = i + 1; j < arrToSort.length; j++) {
    if (arrToSort[j] < arrToSort[minimumNumberIndex]) {
        minimumNumberIndex = j
    }
   }
   let temp : number = arrToSort[i]
   arrToSort[i] = arrToSort[minimumNumberIndex]
   arrToSort[minimumNumberIndex] = temp
}

console.log(arrToSort);

